lesson2/hometask2_Onishchenko.rb

```diff
@@ -0,0 +1,20 @@
+module M
+
+	def self.included(base)
+		def base.class_module_method
+			puts 'class'
+		end
+
+		puts base.methods - Object.methods
```

Comment:
First of all, these two lines are completely equal.
Second, please add an explanation that's kind of methods do you output.